mobile/native/src/trade/position/handler.rs

```diff
@@ -49,50 +50,69 @@ pub async fn trade(filled: FilledWith) -> Result<()> {
 
 /// Fetch the positions from the database
 pub async fn get_positions() -> Result<Vec<Position>> {
-    // TODO: Fetch position from database
+    db::get_positions()
+}
 
-    let dummy_position = Position {
-        leverage: 2.0,
-        quantity: 10000.0,
-        contract_symbol: ContractSymbol::BtcUsd,
-        direction: Direction::Long,
-        average_entry_price: 20000.0,
-        liquidation_price: 14000.0,
-        position_state: PositionState::Open,
-        collateral: 2000,
-    };
+/// Update position once an order was filled
```

Comment:
🔧 Looks like a copy-paste error.

```suggestion
/// Update position once an order was submitted.
```